src/App.tsx

```diff
@@ -1,61 +1,134 @@
 import './App.scss';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { Todos } from './types/Todos';
+import { User } from './types/User';
+
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | null {
+  const findUser = usersFromServer.find(user => user.id === userId);
+
+  return findUser || null;
+}
+
+export const todos: Todos[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [newTitle, setNewTitle] = useState('');
+  const [newUser, setNewUser] = useState(todos);
+  const [userIdCount, setUserIdCount] = useState(0);
+
+  const [errorUser, setErrorUser] = useState(false);
+  const [errorTitle, setErrorTitle] = useState(false);
+
+  const changeTitle = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setNewTitle(e.target.value);
+    setErrorTitle(false);
+  };
+
+  const changeUser = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    setUserIdCount(+e.target.value);
+    setErrorUser(false);
+  };
+
+  const hanldeSubmitNewTodo = (e: React.FormEvent<HTMLFormElement>) => {
```

Comment:
There is a typo in the function name `hanldeSubmitNewTodo`. Consider renaming it to `handleSubmitNewTodo`.